### PR TITLE
[Minor] Advanced tokens for radio component

### PR DESCRIPTION
**Pull request by marcialfps**
New advanced tokens in radio component.

### CODE REVIEW

File: docs/src/pages/themeBuilder/themes/AdvancedTheme.json [hunk 1]
```diff
@@ -683,38 +683,23 @@
     "overlayedProgressBarProgressFontColor": "#FFFFFF"
   },
   "radio": {
+    "circleGutter": "10px",
```
[the remark this answers is not in the document]
Comment (marcialfps) [reply docs/src/pages/themeBuilder/themes/AdvancedTheme.json:686]: It is the space between the label and the circle.

[the remark this answers is not in the document]
Comment (aweell) [reply docs/src/pages/themeBuilder/themes/AdvancedTheme.json:686]: The use of gutter is more appropriate when talking about the spacing between multiple components. In this particular case, I would recommend calling it `circleLabelSpacing` (The best I came up 😓 )

[the remark this answers is not in the document]
Comment (marcialfps) [reply docs/src/pages/themeBuilder/themes/AdvancedTheme.json]: Done in last commit. :v:

**Review by aweell — APPROVED** [review]
:+1:

**Pull request merged**